parquet-hadoop/src/main/java/parquet/hadoop/MemoryManager.java

```diff
@@ -55,7 +55,7 @@ public MemoryManager(float ratio, long minAllocation) {
     minMemoryAllocation = minAllocation;
     totalMemoryPool = Math.round(ManagementFactory.getMemoryMXBean().getHeapMemoryUsage().getMax
         () * ratio);
-    LOG.debug(String.format("Allocated total memory pool is: %,d", totalMemoryPool));
+    LOG.info(String.format("Allocated total memory pool is: %,d", totalMemoryPool));
```

Comment:
Good point, especially when the warning I added when the allocation is changed now includes this information. I've changed this back to debug.